src/General/Alert/Alert.tsx

```diff
@@ -130,12 +130,18 @@ const Alert = ({
   ) : null;
 };
 
-interface Props {
+export interface Props {
+  /** Set Alert's style. */
   type: string;
-  message: React.ReactNode;
+  /** Set message for Alert. */
+  message: string;
+  /** A function to close Alert. */
```

Comment:
I think we should also elaborate on when `onClose` will be triggered. 
Like press which key, or click the close icon, and so on.